ex2/ex2.c

```diff
@@ -1,14 +1,36 @@
-// Write a program that opens the text.txt  file (with the `fopen()` system call) located in this directory 
-// and then calls `fork()` to create a new process. Can both the child and parent access the file descriptor 
+// Write a program that opens the text.txt  file (with the `fopen()` system call) located in this directory
+// and then calls `fork()` to create a new process. Can both the child and parent access the file descriptor
 // returned by `fopen()`? What happens when they are written to the file concurrently?
 
 #include <stdio.h>
 #include <unistd.h>
 #include <stdlib.h>
 
-int main(int argc, char* argv[])
+int main(int argc, char *argv[])
```

Comment:
It isn't actually incorrect to have the asterisk right next to the type declaration. It's purely just a syntax choice.